setup.py

```diff
@@ -23,13 +25,7 @@ def readme():
 def get_version():
     with open(version_file, 'r') as f:
         exec(compile(f.read(), version_file, 'exec'))
-    import sys
-
-    # return short version for sdist
-    if 'sdist' in sys.argv or 'bdist_wheel' in sys.argv:
```

Comment:
these code might be necessary, we need to check whether the delete is safe.

Reply:
I have checked that the delete is safe.